core/src/main/scala-2.12-/cats/instances/all.scala

```diff
@@ -61,4 +61,7 @@ trait AllInstancesBinCompat4 extends SortedMapInstancesBinCompat1 with MapInstan
 
 trait AllInstancesBinCompat5 extends SortedSetInstancesBinCompat0
 
-trait AllInstancesBinCompat6 extends SortedSetInstancesBinCompat1 with SortedMapInstancesBinCompat2
+trait AllInstancesBinCompat6
```

Comment:
Doesn't this break bincompat on 2.11? Only with the recent release candidates, I guess, so maybe it's fine?

Reply:
I would assume so. I think we should only bump mima on actual releases (not including any pre-release).

Reply:
@johnynek Right, but it's still useful to know, so we can decide whether we need to publish new pre-releases of cats-effect or Circe after a Cats pre-release, etc.

Reply:
the plan is to drop 2.11 on master immediately after 2.0.0 release (this evening). So we can add this to the root trait.